src/main/java/uk/gov/dvsa/model/cvs/VTP30Bilingual.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.dvsa.model.cvs;
+
+public class VTP30Bilingual extends VTP30W {
+
+    public String getTestType() {
+        return "PSV";
+    }
+
+    public String getPresentedDocumentNameFail() {
+        return "VTP30";
+    }
+
+    public String getPresentedDocumentNameFailWelsh() {
+        return "VTP30W";
+    }
+
+    public String getVersionNumberFail() {
+        return "1.0";
+    }
+
+    public String getVersionNumberFailWelsh() {
+        return "1.0";
+    }
+
+    public String getRegOrIdHeading() {
+        return "Identification number";
+    }
+
+    public String getRegOrIdHeadingWelsh() {
+        return "Rhif adnabod";
```

Comment:
The same would then go for this, it would be the translation of "Reference number"